src/unix/pipe.c

```diff
@@ -350,6 +335,26 @@ int uv_pipe_chmod(uv_pipe_t* handle, int mode) {
     return r;
   }
 
+  /* Unfortunately fstat on a pipe has a bug on OSX Darwin */
```

Comment:
Nit: I don't think this comment provides enough information outside of the context of this PR. Perhaps at least say that `stat` is used instead of `fstat` due to a bug on macOS."